src/Computation/IConstraints.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Symbolica.Computation;
 
-internal interface IModel : IDisposable
+internal interface IConstraints : IDisposable
 {
     bool IsSatisfiable(IValue assertion);
     BigInteger Evaluate(IValue value);
```

Comment:
Maybe I'm understanding this incorrectly, but would it make sense to rename the `IEnumerable<KeyValuePair<string, string>> Evaluate()` method to `GetModel`? Isn't that what Z2 calls it when you ask for a set of values that satisfy the constraints?

Reply:
Z2 👎 

You might be right but have another look at it on the next PR. It will be clearer then what the various things are. That PR will be quite involved so I was just getting this renaming out of the way because it will be tricky enough to review without all the noise.